src/java.desktop/macosx/classes/sun/java2d/opengl/CGLGraphicsConfig.java

```diff
@@ -55,14 +55,15 @@
 import sun.java2d.pipe.hw.AccelTypedVolatileImage;
 import sun.java2d.pipe.hw.ContextCapabilities;
 import sun.lwawt.LWComponentPeer;
+import sun.lwawt.macosx.CFRetainedResource;
```

Comment:
Looks like lots of changes are not needed on this file

Reply:
Fixed under - JDK-8260706